qa/L0_long_running_stress/stress.py

```diff
@@ -219,6 +226,30 @@ def get_trial(is_sequence=True):
         return np.random.choice(_trials)
 
 
+def count_test_case(test_case_name):
+    # Count the times each test case runs
+    _test_case_count_mutex.acquire()
```

Comment:
We want the threads to send requests as fast as possible, and taking these locks is potentially going to slow things down. Can you instead give each thread their own "test case count" hashes and then you can update without a lock. At the end of the run you can accumulate all the individual thread counts.

Reply:
I see, will modify.